controllers/controllers.js

```diff
@@ -0,0 +1,9 @@
+const {fetchTopics} = require('../models/models')
+
+exports.getTopics = (req, res) => {
+    console.log('in controller')
+    fetchTopics().then((topics) => {    
+        res.status(200).send({topics})
+    })
```

Comment:
Can we just stick a catch block on here please?

Reply:
Hi again John, I have submitted the corrections by pushing them to this branch. I don't think I need to make a new pull request right? I happily will if I need to!